spatialos.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "your_project_name_here",
   "project_version": "1.0.0",
-  "sdk_version": "14.9.0",
+  "sdk_version": "15.0.0",
```

Comment:
We should also update the `README.md` to mention running with `--runtime_version=15.0.0`.